File: 코딩테스트 입문/ㅁ/120911_문자열_정렬하기_(2).cpp
// https://school.programmers.co.kr/learn/courses/30/lessons/120911

#include <string>
#include <vector>
#include <cctype>
#include <algorithm>

using namespace std;

string solution(string my_string) {
    string answer = "";
    vector<char> v;
    for(auto str : my_string) {
        v.push_back(tolower(str));
    }
    sort(v.begin(), v.end());

    for(auto vv : v) {
        answer += vv;
    }
    return answer;
}

// 블로그 : tolower(c++)

File: 코딩테스트 입문/ㅁ/120911_문자열_정렬하기_(2).java
// https://school.programmers.co.kr/learn/courses/30/lessons/120911

import java.util.Arrays;

class Solution {
    public String solution(String my_string) {
        my_string = my_string.toLowerCase();

        char[] charArray = my_string.toCharArray();
        Arrays.sort(charArray);

        return new String(charArray);
    }
}

// 블로그 : toLowerCase()(java), toCharArray()(java)